docs/src/pages/ui/primitives/tabs/index.page.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Tabs
+---
+
+import { Fragment } from '@/components/Fragment';
+
+<Fragment>{({ platform }) => import(`./${platform}.mdx`)}</Fragment>
```

Comment:
TODO: will do a cleanup of all primitives docs pages to use multiple fragments rather than a single one.

Reply:
This will be especially useful if we can share primitive documentation text and swap out the code based on platform, like is done in Authenticator.

Reply:
Yup!